python/lsst/ts/nightreport/shared_state.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+__all__ = ["create_shared_state", "delete_shared_state", "get_shared_state"]
+
+import logging
+import os
+import urllib
+
+import sqlalchemy as sa
+
+from .create_tables import SITE_ID_LEN, create_nightreport_table
+from .database import NightReportDatabase
+
+_shared_state: None | SharedState = None
+
+
+def get_env(name: str, default: None | str = None) -> str:
+    """Get a value from an environment variable.
+
+    Parameters
+    ----------
+    name
```

Comment:
Please add type info to all parameters. In this case
```
name : `str`
```
Note the spaces around the `:`.